linux/sample_driver.sh

```diff
@@ -9,11 +9,11 @@ DIRECTORY="/tmp/csvs"
 
 echo "Directory: $DIRECTORY."
 
-# Loop through every CSV file in the directory and call sendcsv.bat
+# Loop through every CSV file in the directory and call send_csv.sh
 DIRECTORY="$DIRECTORY/*.csv"
 for f in $DIRECTORY; do
 	echo "Uploading: $f."
-	./sendcsv.sh "$f"
+	./send_csv.sh "$f"
```

Comment:
Yeah exactly send_csv would be generated by us.  I think it's a lot to ask clients to generate their own SSH keys and then edit these scripts to include the right usernames and passwords.  With this PR we do that from our end and just send them send_csv.sh